apps/cnp/plum-frontend/preview.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
plum doesn't run on preview as a permanent app only for pull requests, it doesn't make sense to add this at all